proto/regen/ecocredit/v1alpha2/query.proto

```diff
@@ -399,3 +417,81 @@ message QueryAllowedAskDenomsResponse {
   // pagination defines an optional pagination for the response.
   cosmos.base.query.v1beta1.PageResponse pagination = 2;
 }
+
+// Basket defines a credit basket.
+message Basket {
```

Comment:
Should we move this to `types.proto`?

Reply:
it's only being used for queries so I'd rather keep it in `query.proto`